external-crates/move/crates/move-vm-runtime/src/loader/arena.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+// Arena Definitions
+
+use std::mem::MaybeUninit;
+
+use bumpalo::Bump;
+
+pub struct Arena(Bump);
+
+impl Arena {
+    pub fn new() -> Self {
+        Arena(Bump::new())
+    }
+
+    /// SAFETY: it is the caller's responsibility to ensure that `self` is not shared across
+    /// threads during this call. This should be fine as the loader should run in lock-isoltion and
```

Comment:
`lock-isoltion` typo
not sure what lock-isolation means here but essentially all mutable work during "compilation" is on a single thread for a given arena, right?

Reply:
Yep. Updated the comment a bit to indicate that.